README.md

```diff
@@ -11,39 +11,10 @@ The `elixir-omg` repository contains OmiseGO's Elixir implementation of Plasma a
 <!--ts-->
    * [Getting Started](#getting-started)
       * [Service start up using Docker Compose](#service-start-up-using-docker-compose)
-         * [Mac](#mac)
-         * [Linux](#linux)
-         * [Get the deployed contract details](#get-the-deployed-contract-details)
          * [Troubleshooting Docker](#troubleshooting-docker)
-      * [Install on a Linux host &amp; manual start up](#install-on-a-linux-host--manual-start-up)
-         * [Follow the demos](#follow-the-demos)
+       * [Install on a Linux host](#install-on-a-linux-host)
       * [Troubleshooting](#troubleshooting)
```

Comment:
One more to remove here. In the table of content.

Reply:
sloppy murko! tnx!